pkg/providers/amifamily/bootstrap/custom.go

```diff
@@ -15,15 +15,47 @@ limitations under the License.
 package bootstrap
```

Comment:
Can we add some test cases into the `User Data` section of the launch template file (see https://github.com/aws/karpenter/blob/main/pkg/providers/launchtemplate/suite_test.go#L842)